packages/discovery-provider/ddl/migrations/0107_add_email_tables.sql

```diff
@@ -0,0 +1,46 @@
+begin;
+
+-- Store encrypted emails with their metadata
+CREATE TABLE IF NOT EXISTS encrypted_emails (
+    email_id INTEGER PRIMARY KEY,
+    seller_user_id INTEGER NOT NULL,
+    encrypted_email TEXT NOT NULL  -- base64 encoded
+);
+
+COMMENT ON TABLE encrypted_emails IS 'Stores encrypted emails and their metadata for secure communication between sellers and buyers';
+COMMENT ON COLUMN encrypted_emails.email_id IS 'Unique identifier for each encrypted email record';
+COMMENT ON COLUMN encrypted_emails.seller_user_id IS 'User ID of the seller who owns the encrypted email';
+COMMENT ON COLUMN encrypted_emails.encrypted_email IS 'Base64 encoded encrypted email content';
+
+-- Store the encryption keys for sellers and grantees
+CREATE TABLE IF NOT EXISTS email_encryption_keys (
+    key_id INTEGER PRIMARY KEY,
```

Comment:
What is this? Is it necessary? Should the primary key be `seller_user_id`?

Reply:
just gonna swap this with a monotonic primary key `id`

Reply:
I think my comment still stands (see the other comment too) I would rather use the seller_user_id directly instead as the pkey I think